components/atoms/icons/StarCircleIcon/StarCircleIcon.story.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {text} from '@storybook/addon-knobs';
+import {withReadme} from 'storybook-readme';
+import StarCircleIcon from './StarCircleIcon';
+import StarCricleIconReadme from './StarCircleIcon.md';
+
+const stories = storiesOf('Atoms/Icons/StarCircleIcon', module);
+
+stories.addDecorator(withReadme(StarCricleIconReadme));
+
+const defaultProps = () => ({
+  fill: text('fill', '#FFF'),
+  outerFill: text('outer fill', 'Green'),
```

Comment:
```suggestion
  outerFill: text('outerFill', 'Green'),
```

Can we keep the prop name in the knobs table consistent with what's being passed into the component?